lib/db.js

```diff
@@ -202,15 +202,20 @@ class Database extends EventEmitter {
    * Inserts the given docs into a collection.
    * @param {string} collectionName - Name of the collection.
    * @param {(object|Array)} docs - Documents objects to insert.
+   * @param {object} options - options on insert
+   * @param {object} [options.writeConcern] - the write conern options https://mongodb.github.io/node-mongodb-native/4.0/classes/writeconcern.html
    * @returns {Promise} resolves with an object with results, and ops as keys
    * @see {@link http://mongodb.github.io/node-mongodb-native/3.0/api/Collection.html#insertmany}
    * @see {@link http://mongodb.github.io/node-mongodb-native/3.0/api/Collection.html#insertOne}
    */
-  async insert(collectionName, docs) {
+  async insert(collectionName, docs, options) {
+    options = defaults(options, {
```

Comment:
do you really need this defaulting, since its just undefined? (same in the other spots)